spec/views/blog_posts/index_spec.rb

```diff
@@ -34,4 +41,11 @@
     render
     expect(rendered).to have_selector('.item-preview .category-flag', text: blog_posts.first.taxonomy[:blogs].values.first)
   end
+
+  it 'has theme filter' do
```

Comment:
@lbiedinger thanks. I have spec'd both scenarios now, i.e. theme filters enabled and disabled.